templates/cli/lib/questions.js.twig

```diff
@@ -663,7 +658,15 @@ const questionPushChanges = [
     {
         type: "input",
         name: "changes",
-        message: `Would you like to apply these changes? Type "YES" to confirm.`
+        message: `Are you sure you want to apply these changes? (YES/NO)`
+    }
+];
+
+const questionPushChangesFix = [
```

Comment:
what does this variable mean? The name is a bit confusing

Reply:
used `questionPushChangesConfirmation`